templates/zerver/for/open-source.md

```diff
@@ -150,6 +150,11 @@ Efficiently refer to issues or code reviews with notation like `#1234` or
 [custom linkification filter](/help/add-a-custom-linkifier) for
 your organization.
 
+### Open code in external playgrounds.
+
+Configure [external code playgrounds](/help/add-a-custom-playground) in which
+you can easily open and run code occuring in a Zulip message's code block.
```

Comment:
I think my best idea for how to explain this feature is this:

"Instantly transfer content from a [Zulip code block](/help/...) to an [external code playgrounds](/help/...) that can run the code."

Reply:
Typo in Tim's comment -- it should be "playground", not "playgrounds".

+1 for making sure that a user who's never heard of playgrounds can learn what they are by reading the instructions.

Reply:
Sounds great! Also, do we want to stick the same positioning (as in this commit) of where the feature will appear in the list of features in /for/open-source?